src/V3Timing.cpp

```diff
@@ -571,6 +572,7 @@ class TimingControlVisitor final : public VNVisitor {
     void visit(AstScope* nodep) override {
         VL_RESTORER(m_scopep);
         m_scopep = nodep;
+        senExprBuilder = SenExprBuilder{m_scopep};
```

Comment:
I think we want a VL_RESTORER(m_senBuilderr)

Reply:
Can there be scopes in scopes? Then we need the copy constructor.

Reply:
I think they do not next, but assert here if not accepted.